include/seastar/core/shared_ptr_debug_helper.hh

```diff
@@ -62,6 +62,9 @@ public:
         check();
         return _counter--;
     }
+    void migrate_to_this_cpu() {
```

Comment:
Shouldn't you add a test case?

Reply:
Sure, if maintainers ok the general approach.